Library/Homebrew/test/x11_requirement_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "requirements/x11_requirement"
+
+describe X11Requirement do
+  subject { described_class.new }
+
+  describe "#eql?" do
```

Comment:
Isn't this test basically checking if `eql?` is using `hash` to compare the two items? It isn't, by the way, which is odd.